icons/folder-pinia-stores-open.svg

```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32">
+  <path fill="#FFC107" d="M29,12H9.4c-0.9,0-1.6,0.6-1.9,1.4L4,24V10h24c0-1.1-0.9-2-2-2H15.1c-0.5,0-0.9-0.2-1.3-0.5l-1.3-1.1
+	C12.2,6.2,11.7,6,11.3,6H4C2.9,6,2,6.9,2,8v16c0,1.1,0.9,2,2,2h22l4.8-11.2c0.4-1,0-2.2-1.1-2.6C29.5,12.1,29.2,12,29,12z" />
+  <path fill="#FFECB3" d="M29.9,11.8c-0.3-0.5-2.4-0.5-4,0.5c0.1-2.4-1.1-4.8-1.6-4.8c-0.5-0.1-2.2,1.9-2.8,4.2c-1.3-1.4-3.2-1.8-3.6-1.4
+	c-0.5,0.4-0.6,3.5,0.6,5.5c-1.4,1.6-2.3,4-2.3,6.6c0,4.8,2.9,5.8,6.5,5.8s6.5-1,6.5-5.8c0-2-0.5-3.8-1.4-5.3
+	C29.6,15.4,30.3,12.3,29.9,11.8z" />
+</svg>
```

Comment:
The icons do not fit into the grid a bit

https://pixp.lucode.ar/material-extensions/vscode-material-icon-theme/pull/2350